prime-router/src/main/kotlin/cli/tests/TestReportStream.kt

```diff
@@ -958,11 +950,39 @@ abstract class CoolTest {
             allGoodCounties = allGoodReceivers.joinToString(",") { it.name }
         }
 
-        fun initListOfGoodReceiversAndCountiesForUniversalPipeline() {
-            allGoodReceivers = mutableListOf(universalPipelineReceiver1)
-            allGoodCounties = allGoodReceivers.joinToString(",") { it.name }
+        fun initTestDataForUniversalPipeline() {
```

Comment:
Again, Kotlin noob here, let me know if I'm doing anything weird with this.

Reply:
I like this a lot more! Array/Lists and the creation thereof in Kotlin is something that still feels weird to me.